modules/components/pages/inputs/redpanda_migrator.adoc

```diff
@@ -705,6 +708,32 @@ The label of the `redpanda_migrator` output in which the currently selected topi
 
 *Type*: `string`
 
-*Default*: `"redpanda_migrator_output"`
+*Default*: `redpanda_migrator_output`
+
+=== `replication_factor_override`
+
+Use the value specified in the `replication_factor` field when creating topics.
+
+*Type*: `bool`
+
+*Default*: `true`
+
+=== `replication_factor`
+
+Replication factor for created topics. This is only used when `replication_factor_override` is set to `true`.
+
+*Type*: `int`
+
+*Default*: `3`
+
+
+=== `metadata_max_age`
+
+The maximum period of time after which metadata is refreshed.
```

Comment:
What's the significance of this option? We should provide more guidance on when to change this.

Reply:
@mihaitodor. Under what circumstances would we recommend updating this option?

Reply:
I think some users want to have this system pick up newly created topics quicker. See https://github.com/redpanda-data/connect/issues/2879

Reply:
Updated the description to reflect this.

Reply:
Again @mihaitodor. Could you have a quick check to make sure that I have understood this correctly.